db/migrate/20250626214538_create_state_file_archived_intake_access_logs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateStateFileArchivedIntakeAccessLogs < ActiveRecord::Migration[8.0]
+  def change
+    create_table "state_file_archived_intake_access_logs" do |t|
```

Comment:
🤔 i don’t know if we need this because we’ll be handling logging with datadog, but i could be wrong

Reply:
i need to ask Erica about this, we can scrap this table if we want to do event logging like we do for every other event utilizing datadog